packages/editor/src/client/panel/authorEditPanel.tsx

```diff
@@ -140,13 +146,11 @@ export function AuthorEditPanel({id, onClose, onSave}: AuthorEditPanelProps) {
       </Drawer.Header>
 
       <Drawer.Body>
-        <PanelGroup accordion>
```

Comment:
No need to make it as accordion?

Reply:
i removed the accordion because it was of no use. Now it shows everything at once.